Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM python:3.8
+
+# Copy current sources
+COPY . /src
+
+# install bets
+RUN pip install /src
```

Comment:
We should merge all of these `pip` commands into a single `RUN` so that we don't end up with a heap of layers.